src/cmd_line.rs

```diff
@@ -34,6 +36,8 @@ impl Default for CmdLineSettings {
             multi_grid: false,
             maximized: false,
             frameless: false,
+            wayland_app_id: String::from("neovide"),
+            x11_wm_class: String::from("neovide"),
```

Comment:
Is there anything that speaks against exchanging those two to `String::new()`?

Reply:
`String::new()` does not take arguments.

Reply:
Does it need to in our case? We set it via unwrap_or.. later?

Reply:
I mean I tested it before asking the question and it works fine, and seems logical to me to not set it twice, but I am also a novice, so this might be a silly question :) Sorry!

Btw, thats why I wanted to try `unwrap_or_default()` instead of `unwrap_or("neovide")` earlier :)

Reply:
Hmm, makes sense. Let's use String::new.